torch/ao/pruning/_experimental/pruner/README.md

```diff
@@ -1,93 +1,263 @@
-# Intro
+# Structured Pruning
+
+## Intro / Motivation
+
+**pruning** is the process of removing weights / connections in a neural network in order to make it run faster.
```

Comment:
I would perhaps elaborate on this terminology, it kind of assumes the user is familiar how we tend to think about NNs, also weights could mean entire tensors, zeroing out values, or changing dimensionality, a bit ambiguous